client/src/components/Detail.module.scss

```diff
@@ -0,0 +1,77 @@
+.detail {
+  padding: 12px;
+  text-align: left;
+  width: 1084px;
+  display: flex;
+
+  &__col1 {
+    display: flex;
+    height: 100vh;
```

Comment:
This is setting this element to the full view height, which when you add the navbar height to that makes this element unnecessarily high. Giving it `flex: 1` is enough anyway.
```suggestion
```